### ISSUE TITLE
Feat: Implement Theme toggle button Feature

### ISSUE THREAD

**Issue by AnushkaChouhan25** [vishanurag/Canvas-Editor#1146]
### Is there an existing issue raised for this?

- [X] I have searched the existing issues

### Describe the feature

The website currently lacks a theme change option, which would improve accessibility and user experience, in low-light and dark-light environments. 

### Add ScreenShots

N/A

### Component

N/A

### It should be implemented because

N/A

### Would you like to work on this issue?

None

### Terms & Conditions

- [X] I agree to follow the Code of Conduct
- [X] I'm a GSSOC'24 Ext contributor

**Comment by Sarah-Kazi** [comment]
Hi, I'd like to work on this issue. Please assign it to me
